api/src/main/java/org/apache/iceberg/catalog/ViewCatalog.java

```diff
@@ -41,7 +41,8 @@ public interface ViewCatalog {
    *
    * @param namespace a namespace
    * @return a list of identifiers for views
-   * @throws NotFoundException if the namespace is not found
+   * @throws NoSuchNamespaceException if the namespace is not found
+   * @throws IllegalArgumentException if the namespace is invalid for the catalog
```

Comment:
is this true for all catalogs?

Reply:
What is invalid is depends on the catalog impl. 

some of the catalogs like HiveCatalog throws only IllegalArgumentException for empty namespace. So, I would like to cover that option too.